frontend-react/src/components/ManagePublicKey/ManagePublicKey.tsx

```diff
@@ -1,18 +1,131 @@
-import React from "react";
-import { GridContainer } from "@trussworks/react-uswds";
+import React, { useState } from "react";
+import { GridContainer, Icon, SiteAlert } from "@trussworks/react-uswds";
 
 import { AuthElement } from "../AuthElement";
 import { withCatchAndSuspense } from "../RSErrorBoundary";
+import { USLink } from "../USLink";
+import { showError } from "../AlertNotifications";
 import { MemberType } from "../../hooks/UseOktaMemberships";
+import { validateFileType, validateFileSize } from "../../utils/FileUtils";
+
+import ManagePublicKeyUpload from "./ManagePublicKeyUpload";
+
+export const CONTENT_TYPE = "application/x-x509-ca-cert";
+export const FORMAT = "PEM";
+
+function ManagePublicKeySwitchDisplay() {
+    // const [sender, setSender] = useState("");
```

Comment:
Remove?

Reply:
@stephenkao I thought you wanted me to comment out???

Reply:
Oh, duh -- you're right.  @etanb for context, we decided to keep this code commented out for now since we're not going to support this functionality as part of this push.  If/when we can support this, we can uncomment to enable the functionality again.

@penny-lischer you have a comment for the JSX below, but would you mind adding an inline comment here explaining the situation here as well?

Reply:
I think we should remove for now and reference this thread, or have a comment above (hopefully with a ticket) that explains this? 

We've had commented out code with explanation across the codebase before and it makes it hard to maintain. Who knows, we might not work on this for a while and then we'll lose context.